integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep10Tests.kt

```diff
@@ -0,0 +1,62 @@
+package org.stellar.anchor.platform.test
+
+import kotlin.test.assertFailsWith
+import org.stellar.anchor.api.exception.SepNotAuthorizedException
+import org.stellar.anchor.api.sep.sep10.ValidationRequest
+import org.stellar.anchor.platform.*
+import org.stellar.anchor.util.Sep1Helper.TomlContent
+
+class Sep10Tests(val toml: TomlContent) {
+  lateinit var sep10Client: Sep10Client
+  lateinit var sep10ClientMultiSig: Sep10Client
+
+  init {
+    if (!::sep10Client.isInitialized) {
+      sep10Client =
+        Sep10Client(
+          toml.getString("WEB_AUTH_ENDPOINT"),
+          toml.getString("SIGNING_KEY"),
+          CLIENT_WALLET_ACCOUNT,
+          CLIENT_WALLET_SECRET
+        )
+    }
+    if (!::sep10ClientMultiSig.isInitialized) {
+      sep10ClientMultiSig =
+        Sep10Client(
+          toml.getString("WEB_AUTH_ENDPOINT"),
+          toml.getString("SIGNING_KEY"),
+          CLIENT_WALLET_ACCOUNT,
+          arrayOf(
+            CLIENT_WALLET_SECRET,
+            CLIENT_WALLET_EXTRA_SIGNER_1_SECRET,
+            CLIENT_WALLET_EXTRA_SIGNER_2_SECRET
+          )
+        )
+    }
+  }
+
+  fun testMultiSig() {
+    sep10ClientMultiSig.auth()
+  }
+
+  fun testOk(): String {
+    return sep10Client.auth()
```

Comment:
BTW in the future we can use `anchor.auth().authenticate()` method for all of SEP-10 tests